config/samples/hstreamdb.yaml

```diff
@@ -33,6 +48,17 @@ spec:
     replicas: 1
     container:
       name: admin-server
+  console:
+    image: hstreamdb/hstream-console
+    imagePullPolicy: IfNotPresent
+    replicas: 1
+    container:
+      name: console
+      env:
+        - name: HSTREAM_PUBLIC_ADDRESS
+          value: hstreamdb-sample-internal-hserver:6570
```

Comment:
I think it should use value of the gateway endpoint

Reply:
The gateway component is not required, users will not understand why we set a `endpoint` address here. 
I have write a comment for this env config to tell users how to set this env.